Translation/en/Text/Mods/Maker/ao_face.txt

```diff
@@ -367,9 +367,12 @@ M字バングストレート(右)=M-Shaped Straight Bangs (R)
 // chara/koi_ao_face_00.unity3d
 狐の面=Fox Mask
 
+// chara/sak_acc_bo_hair_f_50.unity3d
+SAK中分けデコ出し=SAK middle parted deco
+
 // chara/zeta/sunshineacc.unity3d
-//Sunshine FH 2 - �؂葵���A�V�����g���o���O=
-//Sunshine FH 6 - �؂葵���T�C�h�J�[��=
+Sunshine FH 2 - �؂葵���A�V�����g���o���O=Sunshine FH 2 - ???
```

Comment:
There are likely a wrongly encoded csv file in a mod that's not utf-8

Reply:
It still makes sense to have some sort of translation for these to avoid going through the translator.

Reply:
Sure. Was just pointing out why it was garbage on the left side.